### DISCUSSION TITLE
'Line2D' Object has no property 'nonpositive'

### DISCUSSION THREAD

**Discussion by b291c571** [mboquien/cigale#39] [Q&A]
I noticed that some of my sources were fit after running 'pcigale run' so I ran “pcigale-plots sed” to view the fitted SEDs 
and it keeps stopping me with an Attribute error: Line2D object has no property “nonpositive”. 
First, how do I get around this, and secondly does the command have to run completely for the SED images that were already created before the error to be put into the “out” directory like the manual suggests?
I am just trying to find the figures that were created before it gave me the error. It gives me a runtime warning that more than 20 figures were opened. Disregard the "No valid best fit SED for source #". I am aware that some of the sources were not fit with a model yet. Thank you again. 

![CigaleCodeProb1](https://user-images.githubusercontent.com/70031659/209156016-f0d5aa74-3fca-4f0f-b19a-ea70e3db8a89.png)
![CodeProb2](https://user-images.githubusercontent.com/70031659/209156020-e2838fbc-71cf-4c15-b2b1-65bc158d2692.png)
![CodeProb3](https://user-images.githubusercontent.com/70031659/209156023-abe8f7e1-1464-4bae-9c3d-c1ebbc2d2863.png)

**Comment by Guang91** [comment]
This might be a version issue. 
Could you update your python to >=3.9 and matplotlib to >=3.3, and try again?
see https://github.com/mboquien/cigale/discussions/4
